site/blog/starlingx-release-2-security-features.md

```diff
@@ -7,15 +7,15 @@ category: features-and-updates
 Security on the edge is crucial. Learn more about how the StarlingX 2.0 release helps addressing this challenge. <!-- more -->
 
 
-As we drive compute to the edge of the network many things change and become more interesting and new problems need to be solved. At a minimum the challenges include space and power limitations which then turns into balancing server costs while enabling emerging applications and new transformational use cases at the edge when you take the next steps. However; even though the way compute is delivered at the edge is different, the need for security remains constant or even more critical.
+As we drive compute to the edge of the network many things change and become more interesting and new problems need to be solved. At a minimum you need to face with space and power limitations. Further challenges include balancing server costs while enabling emerging applications and new transformational use cases at the edge when you take the next steps. However; even though the way compute is delivered at the edge is different, the need for security remains constant or even more critical.
 
 Security is always an important consideration in any deployment. It gets more interesting when we no longer have the high level of physical security in the data center where there are retinal scanners at the doors. As we move to the edge we have an environment where there may be less physical security and nodes that are geographically distributed also make maintenance activities difficult.
 
-Within the StarlingX community these are challenges we have recognized and have been working on for a while to provide solutions for. The key approaches start with security enabled at the lowest level of the stack and build up from there. The key challenges we have tackled so far are:
+Within the StarlingX community, these are challenges we have recognized and have been working on solutions for a while. The key approaches start with security enabled at the lowest level of the stack and build up from there. The key challenges we have tackled so far are:
```

Comment:
Let's drop the "for a while" and then I think we are good.

Reply:
Done